src/api/hooks/useIsCouncilMember.ts

```diff
@@ -0,0 +1,10 @@
+import {useAccounts} from 'context/AccountsContext';
+import {useCouncil} from './useCouncil';
+
+export function useIsCouncilMember() {
+  const {accounts} = useAccounts();
```

Comment:
You can use the `networkAccounts` instead of `accounts`.

```suggestion
  const {networkAccounts} = useAccounts();
```